docs/document/content/features/read-write-split/unsupported-items.en.md

```diff
@@ -8,3 +8,4 @@ weight = 3
 1. Data replication between the master and the slave database.
 1. Data inconsistency caused by replication delay between databases.
 1. Double or multiple master databases to provide write operation.
+1. The data for transaction across Master and Slave nodes is inconsitent. In the Master-Savle Replication, the master nodes need to be used for both reading and writing in the transaction.
```

Comment:
> The data for transaction across Master and Slave nodes is inconsitent

maybe better for:

 `The data for transaction across Master and Slave nodes are inconsistent`

> In the Master-Savle Replication, the master nodes need to be used for both reading and writing in the transaction.

maybe better for:

`In the Master-Savle replication model, the master nodes need to be used for both reading and writing in the transaction.`